.github/BOTMETA.yml

```diff
@@ -1159,6 +1159,8 @@ files:
     maintainers: inetfuture mattupstate
   $modules/web_infrastructure/taiga_issue.py:
     maintainers: lekum
+  $tests/an_action.py:
```

Comment:
That's a tough one. I originally wanted to name it something with `module`, but then decided for `action` since also the docs tend to use the term `action` for what gets executed in a task (also there's the good old `local_action`; and we now got `action groups` instead of `module default groups`). Since this test targets less the end-user but more the content developer (on a role/playbook level), I would think that using `action` is the more correct term and also understandable to the audience.

But then, maybe I'm wrong with this... I also don't mind renaming it, but having some more opinions on this would be helpful to determine what to do IMO :)

Reply:
The feedback on IRC convinced me, it's now renamed. Thanks for the suggestion!